package.json

```diff
@@ -241,5 +241,5 @@
       "pre-push": "if git-branch-is main; then yarn pre-push; fi"
     }
   },
-  "packageManager": "yarn@3.1.0"
+  "packageManager": "yarn@3.0.2"
```

Comment:
issue: This might be an unintended consequence of a bad find and replace for lodash, if I read that correctly. This is downgrading yarn by a minor version.

Reply:
Strange, we have 3.0.2 installed but this says to use 3.1.0.  I'm going to replace our 3.0.2 installation with the 3.1.0 version then to keep this setting correct.